### DISCUSSION TITLE
Fully functional example scene might be needed

### DISCUSSION THREAD

**Discussion by HaroldLever** [SpyrexDE/SmoothScroll#54] [General]
Since smooth scroll container is getting more featured and complex, it might need a fully functional example scene to show (or even teach) users how to use it. Also, a fully functional example scene will also be convenient for developers to test it.

Below is my simple idea about new example scene.
![image](https://github.com/SpyrexDE/SmoothScroll/assets/80692930/5ed20718-10cb-4712-9045-7d5194503943)

Moreover, it might need more pages to show each option 's function. Such as `allow_overdragging` 、 `auto_allow_scroll` and so on.

**Comment by SpyrexDE** [comment]
That's a good idea. Design-wise I would prefer to not have everything on the same "page" but use the whole screen space for each example with its explanation and maybe some interactive elements like sliders and toggles to teach the user about the effect each property has on the scroll feeling.
I'd also like to create a UI theme and use non-default fonts to make it look more like an actual modern application that fits the smooth scroll better.
